game/client_core.cpp

```diff
@@ -74,14 +90,41 @@ void ClientCore::renderGameState()
     std::cout << "Players:" << std::endl;
     for (const auto &player : clientState.players)
     {
-        std::cout << "  x: " << player.x << ", y: " << player.y << ", z: " << player.z
-                  << ", orientation: " << player.orientation << ", score: " << player.score << std::endl;
-    }
-    std::cout << "Students:" << std::endl;
-    for (const auto &student : clientState.students)
-    {
-        std::cout << "  x: " << student.x << ", y: " << student.y << ", z: " << student.z
-                  << ", orientation: " << student.orientation << std::endl;
+
+        Window::cube->set_world(player.world);
+        // Print player world matrix
+        for (int i = 0; i < 4; i++)
+        {
+            for (int j = 0; j < 4; j++)
+            {
+				std::cout << player.world[i][j] << " ";
+			}
+			std::cout << std::endl;
+		}
+        std::cout << "  Score: " << player.score << std::endl;
+        
+        std::cout.flush();
     }
+
+    // Don't need students rn...
+    //std::cout << "Students:" << std::endl;
+    //for (const auto &student : clientState.students)
+    //{
+    //    // Print student world matrix
+    //    for (int i = 0; i < 4; i++)
+    //    {
+    //        for (int j = 0; j < 4; j++)
+    //        {
+    //            std::cout << student.world[i][j] << " ";
+    //        }
+    //        std::cout << std::endl;
+    //    }
+    //}
     printf("\n\n");
+
+
+
+    // Render
+    Window::display_callback(window);
+    Window::idle_callback();
```

Comment:
Like above comment about getting input, rendering is done server-side and not on a per-user basis, right? Again, fine for now, just noting